### PR TITLE
update katello for pulp ansible client requirement

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
Requires https://github.com/theforeman/foreman-packaging/pull/3858

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -7,7 +7,7 @@
 %global gem_name katello
 %global prerelease .pre.master
 %global mainver 3.13.0
-%global release 3
+%global release 5
```
Comment (jlsherrill) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: note comparing to the changelog, when release 4 was made this forgot to get bumped, so i'm skipping it here.  Wonder who did that! Such a rookie mistake! ;)

**Comment by evgeni** [comment]
[test rpm]

**Comment by ekohl** [comment]
[test rpm] I suspect repoclosure needed mashing

**Pull request merged**